ops/Cargo.toml

```diff
@@ -14,9 +14,12 @@ serde = { workspace = true }
 evm_arithmetization = { workspace = true, optional = true}
 proof_gen = { workspace = true }
 trace_decoder = { workspace = true }
+tracing = { workspace = true }
+log = { workspace = true }
+env_logger = { workspace = true }
 
 common = { path = "../common" }
 
 [features]
-default = []
+default = ["evm_arithmetization"]
```

Comment:
kind of, currently I need it, because I want to access [pv[0].block_metadata.block_number](https://github.com/0xPolygonZero/zero-bin/pull/48/files#diff-a9623b4036855f41d3c3f54c072a72e91dfa59947c3ab7946c01f0c7ee839059R71)

Reply:
Then we probably don't want it to be optional anymore? Otherwise it would cause issues upon failure if the feature is disabled